### DISCUSSION TITLE
Bert-VITS2-Extra-v2版本运行 train_ms.py 遇到的问题

### DISCUSSION THREAD

**Discussion by pengpengzi** [fishaudio/Bert-VITS2#318] [Q&A] [answered]
'''
(py38_vits) [www@localhost Bert-VITS2-Extra-v2]$ python train_ms.py
加载config中的配置localhost
加载config中的配置10086
加载config中的配置1
加载config中的配置0
加载config中的配置0
加载环境变量
MASTER_ADDR: localhost,
MASTER_PORT: 10086,
WORLD_SIZE: 1,
RANK: 0,
LOCAL_RANK: 0
02-01 22:02:08 INFO     | data_utils.py:65 | Init dataset...
100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 315/315 [00:00<00:00, 99226.87it/s]
02-01 22:02:08 INFO     | data_utils.py:80 | skipped: 0, total: 315
num_worker数值：16
cpu_count数值：0
02-01 22:02:08 INFO     | data_utils.py:65 | Init dataset...
100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 4/4 [00:00<00:00, 48072.25it/s]
02-01 22:02:08 INFO     | data_utils.py:80 | skipped: 0, total: 4
Using noise scaled MAS for VITS2
Use existed model, skip downloading.
INFO:models:Loaded checkpoint 'Data/hla-2/models/WD_0.pth' (iteration 0)
ERROR:models:enc_p.encoder.spk_emb_linear.weight is not in the checkpoint
ERROR:models:dec.cond.weight is not in the checkpoint
ERROR:models:enc_q.enc.cond_layer.weight_v is not in the checkpoint
ERROR:models:flow.flows.0.enc.spk_emb_linear.weight is not in the checkpoint
ERROR:models:flow.flows.2.enc.spk_emb_linear.weight is not in the checkpoint
ERROR:models:flow.flows.4.enc.spk_emb_linear.weight is not in the checkpoint
ERROR:models:flow.flows.6.enc.spk_emb_linear.weight is not in the checkpoint
ERROR:models:sdp.cond.weight is not in the checkpoint
ERROR:models:dp.cond.weight is not in the checkpoint
ERROR:models:emb_g.weight is not in the checkpoint
INFO:models:Loaded checkpoint 'Data/hla-2/models/G_0.pth' (iteration 0)
INFO:models:Loaded checkpoint 'Data/hla-2/models/D_0.pth' (iteration 0)
******************检测到模型存在，epoch为 1，gloabl step为 0*********************
  0%|                                                                                                                                                                      | 0/21 [00:06<?, ?it/s]
Traceback (most recent call last):
  File "train_ms.py", line 837, in <module>
    run()
  File "train_ms.py", line 375, in run
    train_and_evaluate(
  File "train_ms.py", line 496, in train_and_evaluate
    mel = spec_to_mel_torch(
  File "/home/www/Bert-VITS2-Extra-v2/mel_processing.py", line 86, in spec_to_mel_torch
    mel = librosa_mel_fn(sampling_rate, n_fft, num_mels, fmin, fmax)
TypeError: mel() takes 0 positional arguments but 5 were given

'''
这是我的日志，似乎是参数问题，我不太清楚如何修改

**Comment by Stardust-minus** [comment, marked as answer]
pip install librosa==0.9.2
